gramps/gen/filters/rules/citation/_hasnotetype.py

```diff
@@ -0,0 +1,50 @@
+#
+# Gramps - a GTK+/GNOME based genealogy program
+#
+# Copyright (C) 2002-2007  Donald N. Allingham
+# Copyright (C) 2007-2008  Brian G. Matherly
+# Copyright (C) 2008  Jerome Rapinat
+# Copyright (C) 2008  Benny Malengier
+# Copyright (C) 2011        Tim G L Lyons
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+# -------------------------------------------------------------------------
+#
+# Standard Python modules
+#
+# -------------------------------------------------------------------------
+from ....const import GRAMPS_LOCALE as glocale
+
+_ = glocale.translation.gettext
+
+# -------------------------------------------------------------------------
+#
+# Gramps modules
+#
+# -------------------------------------------------------------------------
+from .._hasnotetypebase import HasNoteTypeBase
+
+
+# -------------------------------------------------------------------------
+# "Sources having notes"
+# -------------------------------------------------------------------------
+class HasNoteType(HasNoteTypeBase):
+    """Citations having note of particular type"""
+
+    name = _("Citations having note of particular type")
```

Comment:
```suggestion
    name = _("Citations having a note of type <type>")
```
I wonder if this template is more consistent with other existing rules?
(with a separate PR to make a corresponding change to the existing note type rule)